week3/test_nn.py

```diff
@@ -0,0 +1,21 @@
+import fasttext
+import re
+from nltk.stem import SnowballStemmer
+stemmer = SnowballStemmer('english')
+
+
+def transform_word(name):
```

Comment:
since we use transform function in multiple places, I think we can separate it out as a util fn!